src/components/ModListUpdateBanner.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+import { mixins } from 'vue-class-component';
+import { Component, Watch } from 'vue-property-decorator';
+
+import UtilityMixin from './mixins/UtilityMixin.vue';
+import ManagerInformation from '../_managerinf/ManagerInformation';
+
+@Component({})
```

Comment:
It could be worth moving to the Vue 3 Composition API - I won't make it a blocker but it's something to consider

Reply:
The last time I looked into Composition API I couldn't figure out how to use Vuex store in the new components, and I still don't. So I'd rather not get sidetracked with it now. Unless you have some pointers on how to get it working easily?